### PR TITLE
10v10 Map

### PR DESCRIPTION
**Pull request by warpdragon**

### CODE REVIEW

**Comment by warpdragon** [comment]
This is ready. I will tweak the visuals later, but it works!

File: game/scripts/vscripts/components/boss/bosses.lua [hunk 1]
```diff
@@ -8,7 +10,9 @@ Bosses = {
     "npc_dota_creature_temple_guardian_spawner",
     "npc_dota_boss_tier_6"
   },
-  -- tier 1
```
Comment (chrisinajar) [inline game/scripts/vscripts/components/boss/bosses.lua]: leave the tier comments in so it's more clear. "type 2" doesn't imply "tier 1", this file becomes counter-intuitive to new eyes

**Comment by warpdragon** [comment]
This is ready. I don't know how to alter the duel code in any meaningful way to result in 20 player duels. Otherwise it works.

**Comment by Haganeko** [comment]
How many camps of each type does it have?

**Comment by warpdragon** [comment]
Waiting on some stuff from carlos.

@Haganeko Per side it has 1 ancient, 4 hard, 7 medium, 8 easy

**Comment by Haganeko** [comment]
Yes. If you make it 2 4 6 8 then I don't necessarily have to make different balances for each map

**Comment by warpdragon** [comment]
Done @Haganeko

**Comment by warpdragon** [comment]
Duels are confirmed to work. 10v10 hud is finished. Ready to merge.

**Pull request merged**